DCCpp_Uno/src/PacketRegister.cpp

```diff
@@ -347,10 +353,12 @@ void RegisterList::writeCVBit(const char *s) volatile{
 
   bitClear(bWrite[2],4);              // change instruction code from Write Bit to Verify Bit
 
-  loadPacket(0,resetPacket,2,3);          // NMRA recommends starting with 3 reset packets
-  loadPacket(0,bWrite,3,5);               // NMRA recommends 5 verfy packets
-  loadPacket(0,resetPacket,2,1);          // forces code to wait until all repeats of bRead are completed (and decoder begins to respond)
-
+  loadPacket(0,resetPacket,2,3); // NMRA recommends starting with 3 reset packets
+  loadPacket(0,bWrite,3,5); // NMRA recommends 5 verify packets
+  loadPacket(0,bWtite,3,6); // NMRA recommends 6 write or reset packets for decoder recovery time
+  loadPacket(0,resetPacket,2,1); // Final reset packetcompleted (and decoder begins to respond)
+  loadPacket(0,idlePacket,2,10);
```

Comment:
the idle packet here doesn't make sense after the reset packet. DCC++ should be generating an idle packet anyway if there are no packets to send currently.